CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 ## [Unreleased]
 
+## [0.0.5] - 2018-05-05
+
+### Added
+* Add `test-local-electron` script (\#134) (@vweevers)
```

Comment:
Is escaping `#` necessary if it's not the first character of the line?

Reply:
Apparently not. I just thought it did :)